FitVid/src/Components/Footer/Footer.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "./footer.css";
+import {
+  AiFillHome,
+  AiFillLike,
+  AiOutlineHistory,
+  AiOutlineSearch,
+  MdOutlineExplore,
+} from "../../Assets/icons";
+const Footer = () => {
+  return (
+    <footer className="footer">
```

Comment:
You have written a redundant code here because applying small piece of code to CSS which make it work the way you want .